src/main/java/pacman/DrawComponents.java

```diff
@@ -35,17 +44,20 @@ public void paintComponent(Graphics g) {
         drawGhosts(g);
         drawPacman(g);
     }
+
     public void drawPacman(Graphics g) {
-        ImageIcon pacmanIcon = new ImageIcon("pacman.gif");
-        Image pacmanImg = pacmanIcon.getImage();
-        g.drawImage(pacmanImg, x, y, 30, 30, null);
+        if(pacman.getExist()) {
+            ImageIcon pacmanIcon = new ImageIcon("pacman.gif");
+            Image pacmanImg = pacmanIcon.getImage();
+            g.drawImage(pacmanImg, x, y, 30, 30, null);
+        }
     }
 
     public void drawWalls(Graphics g) { 
```

Comment:
could you change the name of the attribute "g", for a significant name maybe graphic